src/components/common/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,11 @@
+import { PropsWithChildren } from 'react';
```

Comment:
요건 취향인데 type import를 따로 해주는 것도 나름 괜찮은 것 같아요
```suggestion
import { type PropsWithChildren } from 'react';
```